examples/README.md

```diff
@@ -13,7 +13,7 @@ ______________________________________________________________________
 
 ## Lightning Fabric Examples
 
-We show how to accelerate your PyTorch code with [Lightning Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html) with minimal code changes.
+We show how to accelerate your PyTorch code with [Lightning Fabric](https://lightning.ai/docs/fabric/stable) with minimal code changes.
```

Comment:
I saw that we are pointing to LTS in some places and to stable in others.
We should probably just point to LTS everywhere no?

Reply:
good point, so I would say if it is a specific API, then LTS in general let's point to 2.0+